python file>.py

```diff
@@ -0,0 +1,64 @@
+fs_pairs = {'bob': 'bobby',
+            'apple': 'tree',
+            'dog': 'wolf'}
+choice = None
+def choices():
```

Comment:
You should rename to `printChoices` instead. Good job on DRY-ing though.

Reply:
kk